content/developer/testing/develop-a-test-plan.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: practice
+weight: 5
+title: Develop a test plan
+what: >
+  A test plan exists to document how aspects of your tool will be tested. This includes automated and manual tests and how often they will be run.
+why: >
+  Having a test plan enables developers to know what needs to be tested and validated before a release or merge.
+when: Pre-Release
+importance: Medium
+---
```

Comment:
It would be useful to have some additional content here. Main thoughts:

* Our audience may not be familiar with a test plan
* Our teams really only need an explicit test plan once their testing becomes complicated. If your test plan is "run pytest in CI on every push", you probably don't need to add a document to the repo. But once the test plan/policy includes things like "on pushes to main, pytest will run these additional tests" and "prior to release, perform a large-scale benchmark test to ensure that accuracy hasn't degraded," then you really need to document that.